_data/ddc_offline_files_2.yaml

```diff
@@ -83,6 +93,16 @@
 - product: "ucp"
   version: "3.0"
   tar-files:
+    - description: "3.0.12 Linux"
+      url: https://packages.docker.com/caas/ucp_images_3.0.12.tar.gz
+    - description: "3.0.12 IBM Z"
+      url: https://packages.docker.com/caas/ucp_images_s390x_3.0.12.tar.gz
+    - description: "3.0.12 Windows Server 2016 LTSC"
+      url: https://packages.docker.com/caas/ucp_images_win_2016_3.0.12.tar.gz
+    - description: "3.0.12 Windows Server 1709"
```

Comment:
Remove 1709

Reply:
@ollypom is handling the offline bundle updates. Olly could you please remove 1709? Thanks.

Reply:
Thanks. Resolved in latest push